### PR TITLE
[Doc feature][Accounts-Toolkit Manual-2 ]

### PR DESCRIPTION
**Pull request by v-rpatkur**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0a271308c6b39fd2a6dc4b720082e71a975e3dee/msteams-platform/toolkit/accounts.md) | 87 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4aaa8ab2-ecde-4b19-b58c-6bc0dd4f78c8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/62c653a8a773f4dd129cdb5a3b4de6fd674998b1/msteams-platform/toolkit/accounts.md) | 87 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cce05cfe-d613-4d77-a627-a8b7dfae3ccd) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ed895a28f14d587369c529bac16fda29cf4494fd/msteams-platform/toolkit/accounts.md) | 87 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a5a23c14-54bb-4a36-8ef5-ae46e07a575c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/728fc204aab67ece1f67ab5289a28e65bb9c1593/msteams-platform/toolkit/accounts.md) | 87 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1fa6b84f-ecaf-4c92-80ca-bd65e6d8bc22) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/toolkit/accounts.md [hunk 1]
```diff
@@ -53,15 +53,30 @@ You can create for your Visual Studio Professional or Enterprise subscriptions -
 > [!IMPORTANT]
 > During development, you must load your app within your Teams without distributing it. This is known as **sideloading**.
 
-One of the ways to check if you have a Teams account, verify if you can sideload apps in Teams:
+The following are the two different ways to enable sideloading:
```
Comment (zyxiaoyuer) [inline msteams-platform/toolkit/accounts.md:56]: here should be two different way to check/verify whether sideloading permission is enabled. Not to enable sideloading.

File: msteams-platform/toolkit/accounts.md [hunk 2]
```diff
@@ -53,15 +53,30 @@ You can create for your Visual Studio Professional or Enterprise subscriptions -
...
 
-1. In the Teams client, select **Apps** in left bar.
-2. Select **Manage your apps**.
-3. Check whether you can see the option **Upload a custom app** as shown in the image:
+* **Using Visual studio code**
 
-:::image type="content" source="./images/sideload-check.png" alt-text="Diagram that shows to upload a custom app":::
+    1. Open **Visual Studio Code**.
+    1. Select **Teams Toolkit** from left panel.
+    1. Select **Accounts**.
```
Comment (zyxiaoyuer) [inline msteams-platform/toolkit/accounts.md:62]: select ** Accounts** and log in to your M365 account.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/83e814d62e22b0c958355b90b1e79d80a0a7d73c/msteams-platform/toolkit/accounts.md) | 87 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cebff70c-3407-4515-9808-bae4f66f70c8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/70dcdd7cecd25143b91e2ee79ccd03fef9075c2a/msteams-platform/toolkit/accounts.md) | 86 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d0d03644-a47f-4676-b429-d1c4863888c6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8760f53caf12c6cb8bbacfc2ce3448dabaa31d79/msteams-platform/toolkit/accounts.md) | 87 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c83ddded-ca9b-4cbb-bef7-a5a79b21cc8d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b19886584bc56a7db2e239431a8de5ee6b0ec161/msteams-platform/toolkit/accounts.md) | 88 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3d26456c-1929-47c5-9674-6f9571eb9bf9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8d7156133ecfb42a42d53a9ec32861e1228b49c5/msteams-platform/toolkit/accounts.md) | 88 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1fff1fe2-008c-4cc7-a38a-ad940f0992fc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c70288728575fd22508c5193d35200913e789aff/msteams-platform/toolkit/accounts.md) | 88 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/72634354-caa8-4b3f-a7ec-442dfe717602) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/edff7a7a577efbca011377efc4a3a9f64466320f/msteams-platform/toolkit/accounts.md) | 88 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5a16a337-d2fb-4164-b396-821dc2a6b9a7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e832cef385c4547be7848fe74f68dcf3754fe35a/msteams-platform/toolkit/accounts.md) | 88 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/41b4832f-3a3e-4c6f-9b68-6f185a8d1b5d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0fa9d58eacb1953efce4ed52741f83fa247dc317/msteams-platform/toolkit/accounts.md) | 88 | 18 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c2807ef7-5adf-4f0d-a7cf-5dd72b6e0817) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4b28d61f80cb375ee74d1073156d959a65bf67ce/msteams-platform/toolkit/accounts.md) | 86 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/624c99b4-84db-445f-92a1-114180963afe) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by zyxiaoyuer — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/toolkit/accounts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6da71a183d1ece1bbb6575c031ab30b14a59023a/msteams-platform/toolkit/accounts.md) | 86 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3afb70b3-0fbf-4d16-8f59-f82f5d273869) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**